crates/uv/src/commands/project/init.rs

```diff
@@ -803,12 +803,13 @@ impl InitProjectKind {
                 generate_package_scripts(name, path, build_backend, false)?;
             }
         } else {
-            // Create `hello.py` if it doesn't exist
-            // TODO(zanieb): Only create `hello.py` if there are no other Python files?
-            let hello_py = path.join("hello.py");
-            if !hello_py.try_exists()? && !bare {
+            // Create `<package>.py` if it doesn't exist
+            // TODO(zanieb): Only create `<package>.py` if there are no other Python files?
+            let filename = format!("{name}.py");
+            let package_py = path.join(&filename);
+            if !package_py.try_exists()? && !bare {
                 fs_err::write(
-                    path.join("hello.py"),
+                    package_py,
```

Comment:
I think we need the normalized name here so it's a valid module

Reply:
done with hopefully the right normalization